source/config/profileUpgradeSteps.py

```diff
@@ -376,3 +377,17 @@ def upgradeConfigFrom_10_to_11(profile: ConfigObj) -> None:
 			"hidBrailleStandard added to braille display auto detection excluded displays. "
 			f"List is now: {profile['braille']['auto']['excludedDisplays']}"
 		)
+
+
+def upgradeConfigFrom_11_to_12(profile: ConfigObj) -> None:
+	"""Convert reportFontAttributes from a boolean to a choice of speech and/or braille."""
+	try:
+		reportFontAttributes: bool = profile["documentFormatting"].as_bool("reportFontAttributes")
+	except KeyError:
+		log.debug("reportFontAttributes not present in config, no action taken.")
+		return
+	except ValueError:
+		log.debug("reportFontAttributes is not a boolean, no action taken.")
```

Comment:
This case isn't expected right? We should probably log at a higher level

```suggestion
		log.error("reportFontAttributes is not a boolean, no action taken.")
```